src/misc.go

```diff
@@ -501,7 +501,7 @@ var regExtStrMap = map[string]string{
 
 	"docx?": "application/vnd.openxmlformats-officedocument.wordprocessingml.document",
 	"pptx?": "application/vnd.ms-powerpoint",
-	"tsv":   "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",
```

Comment:
According to [Wikipedia](https://en.wikipedia.org/wiki/Tab-separated_values) and [IANA](https://www.iana.org/assignments/media-types/text/tab-separated-values) it is `text/tab-separated-values`. Did you have something else in mind?